packages/kbn-interpreter/package.json

```diff
@@ -12,7 +12,7 @@
     "@kbn/i18n": "1.0.0",
     "lodash": "npm:@elastic/lodash@3.10.1-kibana1",
     "lodash.clone": "^4.5.0",
-    "socket.io-client": "^2.1.1",
+    "scriptjs": "^2.5.8",
```

Comment:
@epixa got rid of this with his CSP PR, so don't think you should add it back.

Reply:
This was a merge conflict. It should be gone now.